Merge Conflict/Assets/Scripts/UiManager.cs

```diff
@@ -192,6 +213,12 @@ private void SwitchMenu(string requestedMenu)
                 AudioManager.Instance.PlayButtonClickSound();
                 break;
 
+            case "Introduction":
+                OpenMenu(_introduction);
+                _currentOpenedMenu = _introduction;
```

Comment:
_currentOpenedMenu = _introduction; is not needed, is already done in OpenMenu()

Reply:
right!